### ISSUE TITLE
Token Introspection Endpoint returns path-based Cell Url even if the Unit is configured per-Cell FQDN mode

### ISSUE THREAD

**Issue by shimono** [personium/personium-core#331]
# Overview
Some attributes of the response payload from Token Introspection Endpoint are path-based Cell Url even if the Unit is configured per-Cell FQDN mode. 

reported by @akiyoshi in Slack

# Details

## Request
Method ： POST
URL　　 ： https://{CELL-NAME}.{UNIT}/__introspect

Header：
Authorization: Bearer {OTHER-CELL-TCAT}
Content-Type: application/x-www-form-urlencoded

BODY：
token={OTHER-CELL-TCAT}

## Response

```
{
    "sub": "https://{UNIT}/{OTHER-CELL-NAME>/#me",
    "aud": "https://{CELL-NAME}.{UNIT}/",
    "iss": "https://{UNIT}/{OTHER-CELL-NAME}/",
    "active": true,
    "exp": 1550140258,
    "iat": 1550136658,
    "client_id": "https://{APP-CELL-NAME}.{UNIT}/",
    "p_roles": [
        "https://{UNIT}/{OTHER-CELL-NAME}/__role/__/admin"
    ]
}
```

# Analysis

Current implementation of Token Introspection Endpoint simply parses the token and returns the token content.  The following two remedy direction can be considered.

1. Change the Token content Cell URL to Per-Cell-FQDN based 
1. Convert path based Cell URL to Per-Cell FQDN in this Endpoint upon returning JSON

**Comment by shimono** [comment]
This bug is fixed at 1.7.5
Root cause is described in issue #334

**Comment by SawamiWataru** [comment]
Released v1.7.5
